tests/net/bridge/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_NETWORKING=y
+CONFIG_NET_TEST=y
+CONFIG_NET_IPV4=n
+CONFIG_NET_IPV6=n
```

Comment:
While this is fine if just bridging is enabled, it might be good to add an additional test to testcase.yaml that enables IPv4 and IPv6 so that we test that all works well in that case too.